bytewriter/memorymappedwriter.go

```diff
@@ -43,11 +43,13 @@ func NewMemoryMappedWriter(loc string, size int) (*MemoryMappedWriter, error) {
 		return nil, fmt.Errorf("Could not initialize %d bytes", size)
 	}
 
-	b, err := syscall.Mmap(int(f.Fd()), 0, size, syscall.PROT_READ|syscall.PROT_WRITE, syscall.MAP_SHARED)
+	b, err := mmap.MapRegion(f, size, mmap.RDWR, 0, 0)
```

Comment:
Why not `mmap.Map`. It is simpler and on unix, will do exactly what is already being done.